src/createReactQueryClient.js

```diff
@@ -0,0 +1,16 @@
+import { QueryClient } from 'react-query';
+
+const cacheTimeMinutes = 5;
+
+const createReactQueryClient = () => new QueryClient({
+  // https://react-query.tanstack.com/guides/important-defaults
```

Comment:
Check this link to get the idea of the [important-defaults](https://react-query.tanstack.com/guides/important-defaults) which I did override here.